Remove land checks in trymerge

clee2000 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #96401
* #96400

Remove all references to land checks (rebase on viable strict in a different branch) since its no longer used.  Adding ciflow/trunk on merge and/or rebasing the entire pr is preferred.

Commit:
Remove land checks in trymerge

[ghstack-poisoned]

Commit:
Update on "Remove land checks in trymerge"


Remove all references to land checks (rebase on viable strict in a different branch) since its no longer used.  Adding ciflow/trunk on merge and/or rebasing the entire pr is preferred.


[ghstack-poisoned]

huydhn commented on .github/scripts/trymerge.py:
```diff
@@ -1603,14 +1453,12 @@ def merge(pr_num: int, repo: GitRepo,
         get_ghstack_prs(repo, pr)  # raises error if out of sync
 
-    check_for_sev(org, project, skip_mandatory_checks)
+    # check_for_sev(org, project, skip_mandatory_checks)
```
Why was this check commented out? Testing?

Commit:
Update on "Remove land checks in trymerge"


Remove all references to land checks (rebase on viable strict in a different branch) since its no longer used.  Adding ciflow/trunk on merge and/or rebasing the entire pr is preferred.


[ghstack-poisoned]

Commit:
Update on "Remove land checks in trymerge"


Remove all references to land checks (rebase on viable strict in a different branch) since its no longer used.  Adding ciflow/trunk on merge and/or rebasing the entire pr is preferred.


[ghstack-poisoned]

clee2000 commented:
@pytorchbot merge
